modules/scanning-images-by-using-secured-clusters.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="scanning-images-by-using-secured-clusters_{context}"]
+= Scanning images by using secured clusters
+
+By configuring delegated image scanning, you can scan images by using the secured clusters instead of the Central services.
+
+A new delegated registry configuration specifies the registries from which you can delegate image scans. For images that Sensor observes, you can use the delegated registry configuration to delegate scans from no registries, all registries, or specific registries. 
```

Comment:
This may be relic from before we had the UI available, I would recommend referring to it as the `delegated scanning configuration` instead of `delegated registry configuration` to more closely resemble the UI:

![image](https://github.com/user-attachments/assets/7a49b4ef-29b8-4261-876b-ebbb99e606eb)